src/vehicle.h

```diff
@@ -708,6 +708,9 @@ class vehicle : public JsonSerializer, public JsonDeserializer
     // <0 means there is no steering installed at all.
     float steering_effectiveness() const;
 
+    /** Returns roughly driving skill level at which there is no chance of fumbling. */
+    float handling_difficulty() const;
```

Comment:
If this is *roughly* perhaps this should return the ceiling as an int as that's effectively what consumers of this function are likely to do?